utils/fhirpath/fhir_path_processor.bal

```diff
@@ -103,6 +103,82 @@ public isolated function evaluateFhirPath(map<json> fhirResource, string fhirPat
 
 }
 
+# Select the resource elements from the given FHIR resource.
+#
+# + fhirPaths - FhirPath expressions for the required elements
+# + fhirResource - Requested fhir resource
+# + return - Returns FHIR resource with the selected elements
+public isolated function selectResourceElements(string[] fhirPaths, map<json> fhirResource) returns map<anydata>|error {
+    map<json> tempFhirResource = fhirResource.clone();
+    string resourceType = check fhirResource["resourceType"].cloneWithType(string);
+
+    map<anydata> elementMap = {
+        "resourceType": {},
+        "meta": {}
+    };
+
+    foreach string fhirPath in fhirPaths {
+        string[] fhirPathTokens = regex:split(fhirPath, "\\.");
```

Comment:
Use the `regexp` language library if you are using Ballerina Update 5 or above version. 
```suggestion
        string[] fhirPathTokens = regexp:split(re `\\.`, fhirPath);
```

Reply:
Updated in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/35eb466e4284e621e7cb661d9d595e4d2eb8eabb